webrender/src/device.rs

```diff
@@ -1169,7 +1202,34 @@ impl Device {
             }
         }
 
-        gl::bind_framebuffer(gl::FRAMEBUFFER, self.default_fbo);
+        // TODO(gw): Hack! Modify the code above to use the normal binding interfaces the device exposes.
+        gl::bind_framebuffer(gl::READ_FRAMEBUFFER, self.bound_read_fbo.0);
+        gl::bind_framebuffer(gl::DRAW_FRAMEBUFFER, self.bound_draw_fbo.0);
+    }
+
+    pub fn blit_render_target(&mut self,
+                              src_texture_id: TextureId,
+                              layer: i32,
```

Comment:
the parameters here need more semantics in their name. While `layer` refers to the source texture layer, the `x, y, width, and height` are for the destination region. Perhaps, we should use something like `dst_rect: Rect<f32>`, where `Rect` is imported from `euclid`?